docs/roles/integrator/exchange-integration.md

```diff
@@ -110,6 +110,390 @@ result of the ones we provided. We expect that, at this stage, most of the contr
 we provide and therefore the potential nondeterminism in contract compilation should not be a concern. Still, we recommend
 taking the first approach whenever possible to support all transfers at once.
 
+**Example of transfer from a lockup contract**
+
+A contract `test.lockup.near` is deployed and we can check its owner by
```

Comment:
Let's replace `test.lockup.near` with `bowen.lockup.near` to reflect a common relationships between owner or lockups, they would be like that
```suggestion
A contract `bowen.lockup.near` is deployed and we can check its owner by
```